src/string/pangram.rs

```diff
@@ -0,0 +1,93 @@
+use std::collections::HashSet;
+
+//*
+// Pangram - sentence that contains all the letters in the alphabet at least once
+// Perfect Pangram - sentence that contains all the letters in the alphabet once (also named perfect heterogram)
+#[derive(PartialEq, Debug)]
+pub enum PangramStatus {
+    NotPangram,
+    Pangram,
+    PerfectPangram,
+}
+//*
+// Fn that checks if the slice is a pangram
+//
+// if you only need one result use is_pangram(str).0 for bool or use is_pangram(str).1 for PangramStatus
+pub fn is_pangram(pangram_str: &str) -> (bool, PangramStatus) {
```

Comment:
Returning `PangramStatus` status should be enough.

```suggestion
pub fn is_pangram(pangram_str: &str) -> PangramStatus {
```